docs/CNVs/CNV-methods.tex

```diff
@@ -829,44 +916,99 @@ \subsection{Generative Model for Read Counts} \label{coverage-model}
 
 \end{itemize}
 
-In approximate mode
-\begin{itemize}
-\item E step: $\vG = \left( \vI + \vW^T \bar{\vPsi}^{-1} \vW \right)^{-1}, \,
-E \left[ \vz_s \right] = \vG \vW^T \bar{\vPsi}^{-1} (\vm_s - \vm), \,
-E \left[ \vz_s \vz_s^T \right] = \vG + E[\vz_s] E[\vz_s]^T$
-
-$\vG$ is $O(D^2 T)$.  $E[\vz_s \vz_s^T]$ is $O(D^2 S)$.  $\vG_s \vW^T \vPsi_s^{-1}$, which is common to all $E[\vz_s]$, is $O(D^2 T)$ after which $E[\vz_s]$ is $O(D T S)$.  $E[\vz_s \vz_s^T]$ is $O(D^2 S)$.  The E step overall is $O(D T S)$.
-
-\item $\vm = \frac{1}{S} \sum_s \vm_s$ is a one-time cost of $O(T S)$.
+The leading cost is a few terms of $O(D^2 T S)$ flops, each with small prefactors, per iteration. Assuming a total prefactor of $10$ and $T = 2 \times 10^5$, $D = 10$, $S = 500$ a full EM iteration costs $10^{11}$ flops in exact mode. On a single 1 GHz core ($10^9$ flops per second) this comes out to roughly 100 or 10 seconds.\\
```

Comment:
With the new way of getting \bar{\vSigma} there is no longer a fast mode so the 10 second version no longer exists.